Bukkit/src/main/java/com/plotsquared/bukkit/listener/EntityEventListener.java

```diff
@@ -315,11 +315,10 @@ public void onPeskyMobsChangeTheWorldLikeWTFEvent(EntityChangeBlockEvent event)
             PlotArea area = location.getPlotArea();
             if (area != null) {
                 Plot plot = area.getOwnedPlot(location);
-                if (plot != null && !plot.getFlag(MobPlaceFlag.class)) {
-                    plot.debug(e.getType() + " could not change block because mob-place = false");
-                    return;
+                if (plot != null && !plot.getFlag(EntityChangeBlockFlag.class)) {
+                    plot.debug(e.getType() + " could not change block because entity-change-block = false");
+                    event.setCancelled(true);
                 }
-                event.setCancelled(true);
```

Comment:
how does that effect hte bhaviour of the MobPlaceFlag?

Reply:
It doesn't, but, this event shouldn't be covered under the mob place flag because it's often fired without mobs. 
E.g. see players jumping on dripleafs. The mob place flag should belong to spots where mobs are actually placed or somewhat of related to, this seems out of place.